exclulde.rb

```diff
@@ -0,0 +1 @@
+p 'test hound'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.